strawberryfields/backends/tfbackend/__init__.py

```diff
@@ -141,6 +141,54 @@
    :members:
 
 """
+import sys
 
 from .backend import TFBackend
 from .ops import def_type as tf_complex_type
+
+try:
+    import tensorflow as tf
+except ImportError:
+    tf_available = False
+else:
+    tf_available = True
+    tf_version = tf.__version__
+
+
+tf_info = """\
+ImportError:
+To use Strawberry Fields with TensorFlow support, version 1.3 of
+TensorFlow is required. This can be installed as follows:
+
+pip install tensorflow==1.3
+
+Note that TensorFlow version 1.3 is only support on Python version
```

Comment:
This stuff seems like it would be a better fit for the documentation. Then perhaps a short error message that directs the user to the docs? Then you don't have to specify in both places.

Reply:
True, it should be added to the documentation as well. My preference is to have both, and keep the verbose error message (with commands to run), even though it is a bit of duplication.